packages/clear-button/src/clear-button.css

```diff
@@ -12,3 +12,7 @@ governing permissions and limitations under the License.
 
 @import url('./spectrum-clear-button.css');
 @import url('./clear-button-overrides.css');
+
+:host {
```

Comment:
Should this be happening on the CSS side?

Reply:
spectrum-css implement a html button tag which comes with a border-box box -model while `sp-clear-button` doesnt actually render a html button element so this property is skipped!